diff_drive_controller/package.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>diff_drive_controller</name>
+  <version>0.0.1</version>
+  <description>Controller for executing joint-space trajectories on a group of joints</description>
+  <maintainer email="allabana@amazon.com">Anas Abou Allaban</maintainer>
+
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <build_depend>controller_interface</build_depend>
+  <build_depend>hardware_interface</build_depend>
+  <build_depend>pluginlib</build_depend>
+  <build_depend>rclcpp</build_depend>
+  <build_depend>rclcpp_lifecycle</build_depend>
+  <build_depend>rcutils</build_depend>
+  <build_depend>realtime_tools</build_depend>
+  <build_depend>geometry_msgs</build_depend>
+  <build_depend>nav_msgs</build_depend>
+  <build_depend>tf2_msgs</build_depend>
+  <build_depend>boost</build_depend>
+
+  <build_export_depend>controller_interface</build_export_depend>
+  <build_export_depend>hardware_interface</build_export_depend>
+  <build_export_depend>rclcpp</build_export_depend>
+  <build_export_depend>rclcpp_lifecycle</build_export_depend>
+  <build_export_depend>geometry_msgs</build_export_depend>
+
+  <exec_depend>nav_msgs</exec_depend>
+  <exec_depend>tf2_msgs</exec_depend>
+  <exec_depend>controller_interface</exec_depend>
+  <exec_depend>hardware_interface</exec_depend>
+  <exec_depend>pluginlib</exec_depend>
+  <exec_depend>rclcpp</exec_depend>
+  <exec_depend>rclcpp_lifecycle</exec_depend>
+  <exec_depend>rcutils</exec_depend>
+  <exec_depend>geometry_msgs</exec_depend>
+  <exec_depend>boost</exec_depend>
```

Comment:
And here as well.

And if the runtime dependencies are different from the build dependencies (header-only libraries fi), it would be good to differentiate those as well.

Reply:
I couldn't find exact documentation for which library of boost accumulators is in, so i assume it's the math one which would be part of `Boost::boost`. We'll see.